docs/3_updating-revanced-manager.md

```diff
@@ -0,0 +1,12 @@
+# 🔄 Updating ReVanced Manager
+
+There are two methods of updating the Manager.
+
+- In the **Dashboard** page, there is an **Updates** section. You'll be able to tap on the **Update Manager** button when an update is available.
+- Head to the [Manager's GitHub releases](https://github.com/revanced/revanced-manager/releases/latest) to manually download and install the latest Manager APK file.
```

Comment:
Use link titles consistently everywhere. Don't link to the same page with the title  "Manager's GitHub releases" and "ReVanced Manager release" for example, chose one.